From 5d3d25d392fc65473055a0087a2903b562ff19a7 Mon Sep 17 00:00:00 2001
From: Evan Quan <evan.quan@amd.com>
Date: Mon, 19 Jun 2017 16:45:33 +0800
Subject: [PATCH 0147/2940] drm/amdgpu: [hybrid] use amdgpu to support CIK
 ASICs

  - this patch is needed only by hybrid staging
  - we may update packing script to add cik_support system config in future.
    Then this commit will be not needed.

Change-Id: Iba5a69fe1db01a290d8b27dff7116f9e534a74ae
Signed-off-by: Evan Quan <evan.quan@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>

Conflicts:
      drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
---
 drivers/gpu/drm/amd/amdgpu/Kconfig      | 8 ++++----
 drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c | 7 ++-----
 2 files changed, 6 insertions(+), 9 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/Kconfig b/drivers/gpu/drm/amd/amdgpu/Kconfig
index e8af1f5e8a79..468a19bfd3d2 100644
--- a/drivers/gpu/drm/amd/amdgpu/Kconfig
+++ b/drivers/gpu/drm/amd/amdgpu/Kconfig
@@ -17,11 +17,11 @@ config DRM_AMDGPU_CIK
 	help
 	  Choose this option if you want to enable support for CIK asics.
 
-	  CIK is already supported in radeon. Support for CIK in amdgpu
-	  will be disabled by default and is still provided by radeon.
-	  Use module options to override this:
+	  CIK is already supported in radeon. If you enable this option,
+	  support for CIK will be provided by amdgpu and disabled in
+	  radeon by default. Use module options to override this:
 
-	  radeon.cik_support=0 amdgpu.cik_support=1
+	  radeon.cik_support=1 amdgpu.cik_support=0
 
 config DRM_AMDGPU_USERPTR
 	bool "Always enable userptr write support"
diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c
index dacc9b7c2526..6b482bc52b77 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c
@@ -100,7 +100,7 @@ int amdgpu_driver_load_kms(struct drm_device *dev, unsigned long flags)
 			dev_info(dev->dev,
 				 "SI support provided by radeon.\n");
 			dev_info(dev->dev,
-				 "Use radeon.si_support=0 amdgpu.si_support=1 to override.\n"
+		"Use radeon.si_support=0 amdgpu.si_support=1 to override.\n"
 				);
 			return -ENODEV;
 		}
@@ -115,10 +115,7 @@ int amdgpu_driver_load_kms(struct drm_device *dev, unsigned long flags)
 		case CHIP_KABINI:
 		case CHIP_MULLINS:
 			dev_info(dev->dev,
-				 "CIK support provided by radeon.\n");
-			dev_info(dev->dev,
-				 "Use radeon.cik_support=0 amdgpu.cik_support=1 to override.\n"
-				);
+				 "CIK support disabled by module param\n");
 			return -ENODEV;
 		}
 	}
-- 
2.17.1

